### DISCUSSION TITLE
Welcome to Flame_Of_OpenSource Discussions!

### DISCUSSION THREAD

**Discussion by PHAGUN-JAIN** [Newbyes/Flame_Of_OpenSource#8] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Sabari2810** [comment]
Hi, I am a full stack developer, I've been developing and designing websites for almost two years now. And i'd like  to be a part of open source projects. Are you currently contributing to any open source projects?

**Reply by PHAGUN-JAIN** [reply]
Hello ! @Sabari2810  
nice to meet you and  thank you for showing your keen interest, at present we are not working on any project, just doing some hacker rank problems and code-chef also, but  we are open to any idea to work on if you have any  however we will be doing something in coming 1 or 2 months. 
please feel free to communicate further if you have anything to share or ask!

**Reply by PHAGUN-JAIN** [reply]
@Sabari2810 also if you are a complete beginner and want to get a taste of contributing to some project you can check out the README of FLAME_OF_OpenSource Repo for guided steps!!
